packages/service-provider-node-driver/package.json

```diff
@@ -59,7 +59,7 @@
   },
   "optionalDependencies": {
     "kerberos": "2.1.0",
-    "mongodb-client-encryption": "^6.1.1"
+    "mongodb-client-encryption": "^6.3.0"
```

Comment:
Seems like `mongodb-client-encryption` was previously bumped but this package was left at the old version; was this intentional?

Reply:
No, I think that's just an oversight (or maybe a bad merge conflict resolve?) 👍